@anthraxx seems that it doesn't work:
```
$ gdb /bin/ls -q
WARNING: Readline services not available or not loaded.
WARNING: The auto-indent feature requires the readline library
pwndbg: loaded 165 commands. Type pwndbg [filter] for a list.
pwndbg: created $rebase, $ida gdb functions (can be used with print/break)
Reading symbols from /bin/ls...(no debugging symbols found)...done.
pwndbg> version
Gdb:      GNU gdb (Debian 7.7.1+dfsg-5) 7.7.1
Python:   2.7.9 (default, Jun 29 2016, 13:11:10)  [GCC 4.9.2]
Pwndbg:   1.0.0 build: 5747cb2
Capstone: 4.0.1024
Unicorn:  1.0.0
pwndbg> set exception-verbose on
Set whether to print a full stacktracefor exceptions raised in Pwndbg commands to True
pwndbg> entry
Temporary breakpoint 1 at 0x4048c5
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Temporary breakpoint 1, 0x00000000004048c5 in ?? ()
Traceback (most recent call last):
  File "pwndbg/pwndbg/commands/__init__.py", line 109, in __call__
    return self.function(*args, **kwargs)
  File "pwndbg/pwndbg/commands/__init__.py", line 200, in _OnlyWhenRunning
    return function(*a, **kw)
  File "pwndbg/pwndbg/commands/context.py", line 86, in context
    result.extend(func())
  File "pwndbg/pwndbg/commands/context.py", line 98, in context_regs
    return [pwndbg.ui.banner("registers")] + get_regs()
  File "pwndbg/pwndbg/ui.py", line 52, in banner
    banner = ljust_colored(banner, width, str(config.banner_separator))
  File "pwndbg/pwndbg/color/__init__.py", line 81, in ljust_colored
    return x + ((remaining // len(char) + 1) * char)[:remaining]
UnicodeDecodeError: 'utf8' codec can't decode bytes in position 54-55: unexpected end of data

If that is an issue, you can report it on https://github.com/pwndbg/pwndbg/issues
(Please don't forget to search if it hasn't been reported before)
PS: Pull requests are welcome
pwndbg> 
```

--- a/pwndbg/color/__init__.py
+++ b/pwndbg/color/__init__.py
@@ -7,6 +7,8 @@ from __future__ import unicode_literals
 
 import re
 
+import six
+
 import pwndbg.memoize
 
 NORMAL         = "\x1b[0m"
@@ -72,9 +74,17 @@ def terminateWith(x, color):
     return re.sub('\x1b\\[0m', NORMAL + color, x)
 
 def ljust_colored(x, length, char=' '):
+    # TODO: workaround until issue #404
+    if six.PY2:
+        x = x if isinstance(x, six.text_type) else x.decode('utf8')
+        char = char if isinstance(char, six.text_type) else char.decode('utf8')
     remaining = length - len(strip(x))
     return x + ((remaining // len(char) + 1) * char)[:remaining]
 
 def rjust_colored(x, length, char=' '):
+    # TODO: workaround until issue #404
+    if six.PY2:
+        x = x if isinstance(x, six.text_type) else x.decode('utf8')
+        char = char if isinstance(char, six.text_type) else char.decode('utf8')
     remaining = length - len(strip(x))
     return ((remaining // len(char) + 1) * char)[:remaining] + x

